metpy/plots/skewt.py

```diff
@@ -770,6 +770,52 @@ def plot(self, u, v, **kwargs):
         u, v = delete_masked_points(u, v)
         return self.ax.plot(u, v, **line_args)
 
+    def plot_layer(self, u, v, p, hgt, bounds, colors, **kwargs):
+        r"""Plot different layers in different colors.
+
+        Plots the wind data on the hodograph colored
+        differently for different height layers. Designed to
+        accept layer bounds in meters AGL, to correspond with
+        storm relative helicity calculations.
+
+        Parameters
+        ----------
+        u : array_like
+            u-component of wind
+        v : array_like
+            v-component of wind
+        p : array_like
+            pressure in hPa
+        hgt: array_like
+            heights from sounding
+        bounds: array_like
+            array of layer bounds, starting
+            at the bottom of the first layer
+            and ending at the top of the last.
+        colors: array_like
+            array of strings containing the
+            colors for each layer of the
+            hodograph
+        kwargs
+            Other keyword arguments to pass to :meth:`matplotlib.axes.Axes.plot`
+
+        Returns
+        -------
+        list[matplotlib.lines.Line2D]
+            lines plotted
+
+        See Also
+        --------
+        :meth:`Hodograph.plot_colormapped`
+
+        """
+        line_args = self._form_line_args(kwargs)
+        u, v = delete_masked_points(u, v)
```

Comment:
Is this actually using height values for the bounds here? I just tried this and it plots the same segments no matter what I set as the norm (or even if I comment out the norm). It appears to just be breaking the hodograph into three arbitrary segments without regard to the heights I give it, but I'm guessing I might just be missing something in my code.